### PR TITLE
Update special titles

### PR DESCRIPTION
**Pull request by kazzarin**
Removed some extra spaces and added more titles. (I don't know if the double quotes will cause problems.)

I'm not sure how different release names and naming schemes are handled currently so I left some titles out.

### CODE REVIEW

**Comment by AncientAzem** [comment]
I would merge this, but the double quotes on line 413 might cause some issues.

File: lib/providers/download/Nyaa/special.json [hunk 1]
```diff
@@ -387,5 +387,32 @@
...
+	"Peeping Life TV: Season 1??":
+	"Peeping Life S1",
+
+	"Miss Monochrome: The Animation 2":
+	"Miss Monochrome S2",
+
+	"Miss Monochrome: The Animation 3":
+	"Miss Monochrome S3",
+
+	"Ore ga Ojou-sama Gakkou ni "Shomin Sample" Toshite Gets♥Sareta Ken":
```
Comment (wopian) [inline lib/providers/download/Nyaa/special.json]: Escape the double quotes 

i.e `"Ore ga Ojou-sama Gakkou ni \"Shomin Sample\" Toshite Gets♥Sareta Ken"`

**Comment by AncientAzem** [comment]
Do what wopian said I will merge the poll request. I would do it myself, but I can't edit you fork.

**Comment by kazzarin** [comment]
Fixed. Thanks @wopian

**Comment by wopian** [comment]
:+1:

**Comment by akyoto** [comment]
Cool, thank you!

**Pull request merged**